homeassistant/components/camera/__init__.py

```diff
@@ -160,7 +160,7 @@ async def async_get_stream_source(hass: HomeAssistant, entity_id: str) -> str |
 @bind_hass
 async def async_get_mjpeg_stream(
     hass: HomeAssistant, request: web.Request, entity_id: str
-) -> web.StreamResponse:
+) -> web.StreamResponse | None:
```

Comment:
Every place that uses this function needs to be updated to handle `None`.

Reply:
I can't find any other usages of it. There's one in `proxy` component but it just overrides `Camera.handle_async_mjpeg_stream` which is processed here.

Reply:
Yup!